cibuildwheel/__main__.py

```diff
@@ -206,6 +212,9 @@ def main() -> None:
     build_verbosity_str = get_option_from_environment(
         "CIBW_BUILD_VERBOSITY", platform=platform, default=""
     )
+    prerelease_pythons = args.prerelease_pythons or get_option_from_environment(
+        "CIBW_PRERELEASE_PYTHONS", platform=platform, default="0"
+    ) not in {"", "0"}
```

Comment:
There is a helper `strtobool` in `cibuildwheel.util` that can parse this. It would also give us case-insensitive true/false matching, which is a bit more friendly to a Python crowd I think.